Dockerfile

```diff
@@ -3,6 +3,9 @@ FROM jenkins/jenkins:2.168
 USER jenkins
 
 COPY plugins.txt /usr/share/jenkins/ref/plugins.txt
+COPY jenkins.yaml /usr/share/jenkins/ref/jenkins.yaml
+
+ENV CASC_JENKINS_CONFIG=/usr/share/jenkins/ref/
```

Comment:
what does mean CASC?

Reply:
ohh configuration as code I guess, strange way to write this acronym

Reply:
That's it, I've no hand on such environment variable.
BTW, I will add a comment for that.